cpp/util/etcd_delete_test.cc

```diff
@@ -221,6 +224,22 @@ TEST_F(EtcdDeleteTest, ErrorHandling) {
 }
 
 
+TEST_F(EtcdDeleteTest, ForceDelete) {
+  vector<pair<string, int64_t> > keys{make_pair("/one", 1),
+                                      make_pair("/two", 0)};
+  SyncTask sync(&pool_);
+
+  EXPECT_CALL(client_, Delete("/one", 1, _))
```

Comment:
Right on, the compiler on my machine was fine with it, the one on Travis wasn't but this did fix it.